.circleci/config.yml

```diff
@@ -64,11 +64,25 @@ step-library:
           git diff --exit-code -- */*/*.lproj
 
   - &install-dependencies
-      run:
-        name: Install Dependencies
-        command: |
-          carthage bootstrap --platform ios --cache-builds --configuration Debug --no-use-binaries --use-netrc
-          carthage outdated --xcode-warnings # prints warnings if there are outdated carthage dependencies
+      unless:
+        condition:
+          equal: [ "12.0.0", << parameters.xcode >> ]
```

Comment:
CircleCI doesn’t much like the fact that I’m referring to a parameter deep down in here:

```
#!/bin/sh -eo pipefail
# Error calling workflow: 'workflow'
# Error calling job: 'xcode-11-examples'
# Arguments referenced without declared parameters: xcode
# 
# -------
# Warning: This configuration was auto-generated to show you the message above.
# Don't rerun this job. Rerunning will have no effect.
false
```

Not sure of the right syntax to express that I want to invoke wcarthage.sh with Xcode 12 but `carthage` otherwise.